tensorflow/compiler/jit/mark_for_compilation_pass_test.cc

```diff
@@ -322,7 +322,7 @@ TEST(XlaCompilationTest, CallXlaDeviceFuncWithResourceOp) {
       MarkForCompilationPassTestHelper::MarkForCompilation(&graph, &flib_def));
   auto clusters = GetClusters(*graph);
 
-  EXPECT_NE(clusters["A"], "");
+  EXPECT_EQ(clusters["A"], "");
```

Comment:
You can delete this test since we will now not compile functions with resource variables as input.